client/src/views/workspaces/WorkspaceHistoryPage.vue

```diff
@@ -364,8 +387,10 @@ async function onPathChange(node: RouterPathNode): Promise<void> {
   }
 }
 
-async function selectAll(selected: boolean): Promise<void> {
-  entries.value.selectAll(selected);
+async function selectAll(): Promise<void> {
+  const selectEntries = !allEntriesSelected.value;
+  entries.value.selectAll(selectEntries);
+  allEntriesSelected.value = selectEntries;
```

Comment:
Regarding this function:
- `selectEntries` variable is not used, you could just flip the value of `allEntriesSelected` and use it in `selectAll`.
- Moreover the variable `allEntriesSelected` only serves as a toggle here, but is not updated if the user manually checks all items in the history. There is already the `allSelected` variable present in this view which does just that, so you can just call `entries.value.selectAll(!allSelected.value)`.
- This function becomes only this call above, so you can delete it and make the following changes in the template (see next comments)